resources/wikia/modules/iframeWriter.js

```diff
@@ -25,7 +25,7 @@ define('wikia.iframeWriter', [
 		var code = iframeHeader + iframeStyle + params.code + iframeFooter,
 			iframe = doc.createElement('iframe');
 
-		iframe.frameborder = 'no';
+		iframe.frameBorder = 'no';
```

Comment:
You're gonna hate I'm using emoticon with text :japanese_ogre: 

:camel:-case - I see, good catch if it fixes something. I hope it doesn't break anything. It's funny because both HTML and HTMLIFrame documentation on Mozilla pages mentions `frameborder`:
* https://developer.mozilla.org/en/docs/Web/HTML/Element/iframe
* https://developer.mozilla.org/en/docs/Web/API/HTMLIFrameElement

But hated [w3school](http://www.w3schools.com/jsref/dom_obj_frame.asp) and [w3c docs](http://www.w3.org/TR/DOM-Level-2-HTML/html.html) tells about `frameBorder`.